cpp/include/cudf/io/orc.hpp

```diff
@@ -354,6 +354,9 @@ class orc_reader_options_builder {
  *  auto result = cudf::read_orc(options);
  * @endcode
  *
+ * Note: Support for reading files with struct columns is currently experimental, the output may not
+ * be correct in all cases.
```

Comment:
```suggestion
 * be as reliable as reading for other datatypes.
```
maybe like that which I think fairly indicates that it's a work in progress. I think the original wording sounds a little more like it's untested, which wouldn't be accurate by any means, it's more of a corner case issue now.